### ISSUE TITLE
Change saved filters

### ISSUE THREAD

**Issue by totti4ever** [jonaswinkler/paperless-ng#58]
Maybe I understand the idea wrong but I would be kind of stuck once I saved a view. When I open it again, I cannot change the filters, the sort order or anything else. This is bad from my point of view for two reasons:
1. You have to start from scratch in order to just adapt a small thing
1. you cannot use a dashboard for further drilling down into your documents

Idea would be to maybe 
1. just use the normal controls for adding/changing filters and order
1. add some form of save / save as option

**Comment by jonaswinkler** [comment]
Agreed. The idea was to have the documents view for generic filtering, and the saved views for anything predefined.

I thought about adding editing abilities to the settings, but this is the way to go.

What should happen if you were to go from the documents list to a saved view, mess with the filter settings, and then go back to the documents list? Should that still bring up the last filter used for the document list?

**Comment by totti4ever** [comment]
1. One idea would be that clicking on documents list always resets,
1. the other most probably what you said: switching to the documents list keeps the (current and changed) settings
1. or another approach could that you just
	1. switch to documents list, settings are kept and there you can save/overwrite existing views
	1. have a button like 'switch to documents list and keep settings'

Okay, I just figured out that 3.1 is exactly what is happening right now, right?. Quite easy use, once you know about it. So maybe a button _Transfer filters to documents list_ would make it more explicit and transparent?

What still isn't caught is saving a filter under the same name as an existing one - now I have two views called _test_ :-)

**Comment by jonaswinkler** [comment]
Alright, I think I got this figured out without any buttons getting in the way.

- The document list will always remember how it was displayed.
- Going to a saved view will always load that saved view, so it forgets whatever you changed.

This is new:

- The saved view have the usual controls.
- The "Save As" action will save the current view settings as a new view, no matter where you are.
- The "Save" action will be available when a saved view is displayed and persists any changes you made.

I've got that working over here like that and it works out pretty good.
